app/src/App.tsx

```diff
@@ -0,0 +1,69 @@
+import axios from "axios";
+import { useState } from "react";
+import "./App.css";
+
+declare global {
+  interface Window {
+    Razorpay: any;
```

Comment:
can you alternatively use a package like https://www.npmjs.com/package/react-razorpay that supports typing & react integration